server/database/schemas/users.py

```diff
@@ -1,28 +1,44 @@
 from pydantic import BaseModel
 from uuid import UUID
+from datetime import datetime
 
 
 class UserBase(BaseModel):
     """Common user data shared by all schemas"""
 
-    email: str
-    username: str
+    email: str | None = None
+    username: str | None = None
 
 
 class UserGet(UserBase):
     """User data which can be read"""
 
     id: UUID
-    display_name: str
-
-    class Config:
-        orm_mode = True
+    display_name: str | None = None
```

Comment:
We've discussed that if user don't have a display_name, then it becomes an email value. If it is still like this, then `display_name` isn't a very good naming - it hides transformation of it

Reply:
As we discussed privately, it was a misunderstanding